code/__DEFINES/flags.dm

```diff
@@ -99,19 +99,20 @@ GLOBAL_LIST_INIT(bitflags, list(1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 204
 #define MOB_SPAWN_ALLOWED			(1<<4)
 /// If megafauna can be spawned by natural random generation
 #define MEGAFAUNA_SPAWN_ALLOWED		(1<<5)
+/// Are you forbidden from teleporting to the area? (centcom, mobs, wizard, hand teleporter)
+#define NOTELEPORT (1<<6)
```

Comment:
![Screenshot 2024-11-10 152653](https://github.com/user-attachments/assets/ff1a7aab-0e55-4e39-99c2-bd4dcd0c2189)

![Screenshot 2024-11-10 152701](https://github.com/user-attachments/assets/27ad8851-241e-44f1-984b-809a3db38057)

This flag may be unecessary as we already have a tiered var on areas for checking if a mob can teleport there

Reply:
You are absolutely right bestie